LabTools/CoreWidgets/ScriptWidget.py

```diff
@@ -13,40 +12,62 @@
 if USE_PYQT5:
 
     from PyQt5.QtWidgets import (QWidget, QLabel, QLineEdit, QPushButton,
-                                 QFileDialog, QHBoxLayout, QApplication)
+                                 QFileDialog, QHBoxLayout, QApplication, QComboBox, QSizePolicy)
 
 else:
 
     from PyQt4.QtGui import (QWidget, QLabel, QLineEdit, QPushButton,
-                             QFileDialog, QHBoxLayout, QApplication)
+                             QFileDialog, QHBoxLayout, QApplication, QComboBox, QSizePolicy)
+
+try:
+    from pathlib import Path
+    PATHLIB=True
+except:
+    PATHLIB=False
 
 
 from LabTools.IO import IOTool
+import os
 
+LEGACY_SCRIPTWIDGET = False
 
 class ScriptWidget(QWidget):
-    """widget with a lineEdit and a browse button"""
+    """widget with a dropdown and a browse button"""
 
-    def __init__(self, parent=None):
+    def __init__(self, parent=None, LEGACY = None):
         super(ScriptWidget, self).__init__(parent)
 
+        if LEGACY is None:
+            self.LEGACY = LEGACY_SCRIPTWIDGET
+        else:
+            self.LEGACY = LEGACY
```

Comment:
```suggestion
            self.LEGACY = legacy
```

Reply:
@zackorenberg - you can "add suggestion to batch" with the lower right button under my suggestion to bind the suggestion together :)